<launch>
<node pkg="sas_robot_driver_denso" type="sas_robot_driver_denso_node" name="cobotta_2">
<rosparam file="$(find sas_robot_driver_denso)/launch/sas_robot_driver_denso_cobotta_example.yaml" command="load"/>
</node>
</launch>
